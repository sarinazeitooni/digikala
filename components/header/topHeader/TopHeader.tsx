import React from 'react';
import style from './scss/top-header.module.scss';
import messages from "../messages/messages";
import SearchSharpIcon from '@material-ui/icons/SearchSharp';
import ShoppingCartOutlinedIcon from '@material-ui/icons/ShoppingCartOutlined';
import PersonOutlineOutlinedIcon from '@material-ui/icons/PersonOutlineOutlined';
interface props {
    logo : string
}
const TopHeader = ({logo} : props)=>{
    return(
        <div className={style['top-header-container']}>
            <div className={style['account-manager-container']}>
                <button className={style['sign-in-btn']}>{messages.signIn} <PersonOutlineOutlinedIcon/></button>
                <span className={style['vertical-line']}/>
                <ShoppingCartOutlinedIcon/>
            </div>
            <div className={style['input-logo-container']}>
                <img className={style['logo-image']} src={logo}/>
                <div className={style['search-input-container']}>
                    <SearchSharpIcon/>
                    <input className={style['search-input']} placeholder={messages.inputPlaceHolder} type="text"/>
                </div>
            </div>
        </div>
    )
};
export default TopHeader;