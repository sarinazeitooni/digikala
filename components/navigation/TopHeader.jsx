import React from 'react';
import style from './scss/top-header.module.scss';
import messages from "./messages/messages";
const TopHeader = (({icon})=>{
    return(
        <React.Fragment>
            <div className={style['top-header-container']}>
                <div>
                    <img src={icon} alt='digikala-icon'/>
                    <div className={style['input-container']}>
                        <input placeholder={messages.placeholder} type='text'/>
                    </div>
                </div>
                <div>
                    <button className={style['entry']}>{messages.enter}</button>
                    <span className={style['vertical-line']}/>
                    icon
                </div>
            </div>
        </React.Fragment>
    )
});
export default TopHeader;