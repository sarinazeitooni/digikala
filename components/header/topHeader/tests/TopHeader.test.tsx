import React from 'react'
import {shallow} from 'enzyme';
import TopHeader from "../TopHeader";
describe('TopHeader component', () => {
    it('should render without throwing an error', ()=> {
        const wrapper = shallow(<TopHeader/>);
        expect(wrapper).toMatchSnapshot();
    })
});