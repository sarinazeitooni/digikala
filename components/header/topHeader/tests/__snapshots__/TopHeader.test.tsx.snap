// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopHeader component should render without throwing an error 1`] = `ShallowWrapper {}`;
